src/main/java/org/broadinstitute/hellbender/engine/filters/OverclippedReadFilter.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.cmdline.Argument;
+
+/**
+ * Filter out reads that are over-soft-clipped
+ *
+ * <p>
+ *     This filter is intended to filter out reads that are potentially from foreign organisms.
+ *     From experience with sequencing of human DNA we have found cases of contamination by bacterial
+ *     organisms; the symptoms of such contamination are a class of reads with only a small number
+ *     of aligned bases and additionally many soft-clipped bases.  This filter is intended
+ *     to remove such reads. Consecutive soft-clipped blocks are treated as a single block
+ * </p>
+ *
+ */
+public final class OverclippedReadFilter implements ReadFilter{
```

Comment:
ReadFilters are Serializable - so they must have the magic `serialVersionUID` field